### PR TITLE
ENH: merge gradients convert

### PR DESCRIPTION
**Pull request by AntoineTheb**
Merge gradients conversion scripts into one. "Legacy" scripts now point to the same script, `scil_gradients_convert.py`.

### CODE REVIEW

File: scripts/scil_visualize_bundles.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def subsample(list_obj):
             color = subsample(
                 tractogram_gen.data_per_point[args.color_from_points])
         elif args.uniform_coloring:  # Assign uniform coloring to streamlines
-            color = tuple(np.asarray(args.uniform_coloring) / 255)
```
Comment (AntoineTheb) [inline scripts/scil_visualize_bundles.py:180]: Woops, this should not be included.

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-854/1/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-854/2/

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**